lab_8_llm/main.py

```diff
@@ -43,6 +35,18 @@ def obtain(self) -> None:
         Raises:
             TypeError: In case of downloaded dataset is not pd.DataFrame
         """
+        self._raw_data = load_dataset(self._hf_name,
+                                      split='train').to_pandas()
+
+    @property
+    def raw_data(self) -> DataFrame:
```

Comment:
And this?

Reply:
like previous project lab_7, without it mypy tests crash

Reply:
See the comment above

Reply:
did it!